src/Commands/BuildToolsBase.php

```diff
@@ -670,15 +671,15 @@ protected function redactMetadata($metadata, $keys_to_redact)
     /**
      * Determine the command to use to install the site.
      */
-    protected function getInstallCommandTemplate($composer_json)
+    protected function getInstallCommandTemplate($composer_json, $upstream)
     {
         if (isset($composer_json['extra']['build-env']['install-cms'])) {
             return $composer_json['extra']['build-env']['install-cms'];
         }
-        // TODO: Select a different default template based on the cms type (Drupal or WordPress).
-        $defaultTemplate = 'drush site-install --yes --account-mail={account-mail} --account-name={account-name} --account-pass={account-pass} --site-mail={site-mail} --site-name={site-name}';
-
-        return $defaultTemplate;
+        if ($upstream == 'empty-wordpress') {
```

Comment:
We'll need to find a different way to detect WP or Drupal. There are plenty of people out there that use Build Tools with WP upstreams that aren't `empty-wordpress`.

Reply:
Actually, this might not be accurate. One moment...

Reply:
Yup, nevermind here. `autoDetectUpstream()` returns a fixed list.

Reply:
version-tool is a more accurate way to find the framework type.

Reply:
> Build tools expects this in your [WordPress project template](https://github.com/pantheon-systems/example-wordpress-composer/blob/master/composer.json#L53)
>
Yep, I saw that through reading the function - not documented anywhere I can see - but figured a default WP and a default Drupal option covers the basics better, especially with the comment there.

re: using version-tool or something. That fixed list from autoDetectUpstream does seem a bit weak, but figured as it was being used elsewhere that was the most minimal change to be fully functional.

re: next steps, would you suggest switching to the composer.json method and not setting a default? (if so, want a documentation PR about that?). Or do you think having a default is a good direction?